### DISCUSSION TITLE
Currently Available Skins - An overview

### DISCUSSION THREAD

**Discussion by ogami1972** [Drewol/unnamed-sdvx-clone#596] [General]
Hello!

The currently available skins are reviewed here. All have some issues that I wanted to document here for reference.  IMHO, there are NO 2nd party skins currently available that are both fully functional and visually appealing. Some of the skins have inappropriate splash screens or malformations of the game's title, so use discretion in your testing.

Before I start, I'd like to thank every person who ever made a skin for the game. regardless of my findings, having additional skins is greatly appreciated. This post is not meant as criticism, but rather simply a resource for new players.

These findings and opinions are solely my own, and have a lot to do with the capabilities of my system, but I believe they may be of some use.

my rig: i-5 4440, 2gb RX550, 12gb RAM

**Liqidwave 1.5.0** : Hands down the prettiest skin. This skin uses the most resources. Just loading the song selection screen will consume 5-6gb of RAM, and 60fps will be difficult to achieve without turning aliasing way down or off entirely. Playing one chart will consume an additional 1-2gb of RAM that will not be returned to the system after the chart is completed. This results in the system freezing or shutting down after 5-7 songs. This issue has been confirmed by skin developers. 

**Liqidwave 1.4.1** : Similar to 1.5 but uses less to start with (hovers around 2-3gb). Sadly, this skin does NOT support Permissive gauge (or blastive either, probably), so it is limited in functionality, and probably considered outdated.

**Heavenly Express** : Similar to LW 1.4.1 in it's resource hogging and leakage, but much worse in it's limited functionality. Does not offer challenges or optional gauges at this time. However, rumor has it that an update is being worked on.

**Experimental Gear 0.2.2** : Much better resource usage, but still leaks. Challenge results screen causes game to crash. Release does not have hit animations or Nemsys feedback, which is not the end of the world, but is important eye candy. Currently harder to track down. IMO, this is the skin to watch. A working challenge results, hit animations and proper transition screens will make this a winner. 

**Nightfall** : Ultra-minimal. I have not used this skin, but in slight testing, I found it to be very light and fast. Great low-end skin. If you run on low-end equipment, or just want a simple experience, this is probably your best skin, overall.

**Lasergame, SDVX IV, Nero, Enobywave** : I have little experience with these. Most seemed minimal, and landscape-oriented. Additionally, I do not believe these skins are currently maintained.

That's all I got. I hope someone finds this useful, and I will update as necessary.

**Comment by parakeet-live** [comment]
Where would you find Experimental Gear?

**Comment by ogami1972** [comment]
Hello!

Discord, though it took me some research to find it. 

I started with the public invite to "SDVX Community", found on the K-Shootmania subreddit.  This discord server is a busy place and a bit silly. It can be overly focused on SDVX, imo, and hostile to new players who aren't clearing 16+ charts. However, there you will find links to invites to the exceed gear server. If you look long enough, you will find the actual USC server, which for some reason is not publicized in any meaningful way. 

The community is friendly enough in these places, I'm just not sure of the protocol of sharing a bunch of discord invites. Instead, I'll just share the skin directly, lol.

[USC Skin : Experimental Gear 0.2.2](https://drive.google.com/file/d/1BvbqklvY2zK0Jei_9xiXxKYZ3UGrmcpk/view?usp=sharing)

**Reply by parakeet-live** [reply]
Thank you! 👍🏽

**Comment by ogami1972** [comment]
One last thing (Bonus Skin at end of long story):

I play, I believe, differently from other players. 

I have never seen or have any experience with SDVX. I found USC because I'm a shmup player and had been collecting the touhou games, which resulted in a Tokaku video showing up in my youtube recommendeds. The wife and daughter love rhythm games, and I thought the game looked cool, and had a little extra cash, so we bought a controller and dove in, building a dedicated USC system. We play exclusively "local", and are hoping to one day build an actual USC cabinet.

Because of this mindset, we don't use multiplayer. Additionally, I act as an "admin" of sorts, managing player requests for profile adjustments, skin options, and chart/course importing. For these reasons "Multiplayer" and "Nautica" aren't useful to us on every play period. Once a week,  I will load the default skin and visit nautica to pull any new charts, then reload the custom skin I share below.

Players use a special version of the EG skin that only offers Normal Play, Challenges, "Profile Selection" (which is the settings menu) and Exit. Additionally, I have set custom splash images and the "Vivid Wave" nemsys to match our controller. **THIS SKIN IS NOT SUPPORTED BY TEAM EXCEED.** However, It DOES have hit animations, and it does have a working challenge result screen (it's the default, but it works). I call this the "locked down" version.

The way we use this skin is detailed in the discussion "My Current Setup", feel free to ask questions. 

[USC Skin: EG Locked Down](https://drive.google.com/file/d/1dajamrTfQAoHsVe6J08Q7h-INZhFgKDQ/view?usp=sharing)

**Comment by Hoshikara** [comment]
my skin has had portrait support for [quite some time now](https://github.com/Hoshikara/Nightfall/blob/main/CHANGELOG.md#110---may-17-2021), even though preview material does not show such

**Reply by ogami1972** [reply]
Hello!

My apologies, amended.

**Comment by 12beesinatrenchcoat** [comment]
You mention LiquidWave 1.5.0 — where did you find it? I only have LiquidWave 1.4.1, and it seems all other traces of the skin have been erased from the internet.

(I also personally have had decent experiences with LiquidWave, but yeah, ymmv! :3)

**Comment by ogami1972** [comment]
Hello!

LW 1.5.0 Can be found on the "SDVX Community" discord channel. I think Dreamy also hangs out on the USC channel, but I've only seen releases on the former.

Strangely, I seemed to have trashed my copy, which is odd, as I never throw out anything. 

However, I do have a special, UNSUPPORTED version of this skin that has something called "valk effects". This was modified by someone involved with the EG skin. I have not tested this skin at all. Let us know what you think of it.

[USC Skin: Liqidwave 1.5 w/ Valk Effects](https://drive.google.com/file/d/1YMzSHQy1KkcL6Y7loPd2cxH5Tw1JrUM1/view?usp=sharing)
